### PR TITLE
integrated circuit creation timespan

### PR DESCRIPTION
**Pull request by camotts**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Release 0.25.6
```
Comment (plorenz) [inline CHANGELOG.md:1]: 0.25.5 isn't out yet, so this can go below in the 0.25.5 section

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+* Enhancement: Updated Control Channel to use new heartbeat logging mirroring Links in Release `0.25.0`
+* Enhancement: Added Circuit Creation Timespan which denotes how long the fabric took to construct a requested circuit.
```
Comment (plorenz) [inline CHANGELOG.md:4]: Can you add a quick example of what the JSON output looks like so ops knows what to expect

File: CHANGELOG.md (hunk 1 above)
Comment (camotts) [reply CHANGELOG.md]: Must have selected the wrong source branch when checking the changelog, moved to 0.25.5

File: CHANGELOG.md (hunk 2 above)
Comment (camotts) [reply CHANGELOG.md:17]: Added an example json output

**Review by plorenz — APPROVED** [review]

**Pull request merged**